### PR TITLE
add init setup pg16

### PR DESCRIPTION
**Pull request by idanovinda**
- Update `PGVERSION`/`PGOLDVERSIONS` arg default values
- Skip not-yet-available-for-16 extensions installation
  - plproxy
  - partman
  - pglogical
  - pglogical-ticker
  - plpgsql-check
  - pgl-ddl-deploy
  - timescaledb
  - plantuner
  - plprofiler

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ARG BASE_IMAGE=ubuntu:22.04
-ARG PGVERSION=15
-ARG TIMESCALEDB="1.7.5 2.3.1 2.11.1"
+ARG PGVERSION=16
+ARG TIMESCALEDB=""
```
Comment (hughcapet) [inline postgres-appliance/Dockerfile:3]:
```suggestion
ARG TIMESCALEDB="1.7.5 2.3.1 2.11.1"
```

File: postgres-appliance/Dockerfile [hunk 2]
```diff
@@ -48,7 +48,7 @@ ARG TIMESCALEDB
 ARG TIMESCALEDB_APACHE_ONLY=true
 ARG TIMESCALEDB_TOOLKIT=true
 ARG COMPRESS
-ARG PGOLDVERSIONS="10 11 12 13 14"
+ARG PGOLDVERSIONS="10 11 12 13 14 15"
```
Comment (hughcapet) [inline postgres-appliance/Dockerfile:51]:
```suggestion
ARG PGOLDVERSIONS="11 12 13 14 15"
```

File: postgres-appliance/Dockerfile (hunk 1 above)
Comment (hughcapet) [reply postgres-appliance/Dockerfile:3]: and skip for 16

**Comment by jopadi** [comment]
👍

**Comment by idanovinda** [comment]
👍

**Pull request merged**